NitroxClient/MonoBehaviours/Gui/MainMenu/JoinServer.cs

```diff
@@ -33,6 +33,11 @@ public class JoinServer : MonoBehaviour
         private PlayerPreferenceManager preferencesManager;
         public string ServerIp = "";
         public int ServerPort;
+        private string serverPassword = "";
+        private Rect serverPasswordWindowRect = new Rect(Screen.width / 2 - 250, 200, 500, 200);
+        private bool shouldFocus;
+        private bool showingPasswordWindow;
+        private bool passwordEntered;
```

Comment:
Minor: 
- Is it possible to cleanup all of these ? Regroup static, public, private, ...
- string.Empty also